tests/run-make/compressed-debuginfo-zstd/rmake.rs

```diff
@@ -0,0 +1,42 @@
+// Checks debuginfo compression both for the always-enabled zlib, and when the optional zstd is
+// enabled:
+// - via rustc's `debuginfo-compression`,
+// - and via rust-lld's `compress-debug-sections`
+
+//@ needs-llvm-zstd: we want LLVM/LLD to be built with zstd support
```

Comment:
The ignore reason has an additional space, but that's a detail:
```
ignored, ignored if LLVM wasn't build with zstd for ELF section compression ( we want LLVM/LLD to be built with zstd support)
```

Reply:
Ugh! I will look into where this concat appears. I cannot avoid spaces after a colon 😱 

Could you remind me where we store the list of all ignore reasons so I can quickly spot if just stripping would not regress any other ignore reasons?

Reply:
Umm, I have no idea :sweat_smile: We have some test with a list of all ignore reasons?

Reply:
Aren't they colocated with the list of ignored tests? And that similar list was used by pietro for the ferrocene reports which contained the ignore reasons?

Reply:
I created a list of currently ignored tests by running some Python scripts on our CI logs, I'm not aware of a location that actually stores them, tbh. I can try to generate that list, if you want, although I'm not sure if we also store the ignore reasons on CI (but I think that we do).

Reply:
don't worry about it :) I'll strip whitespace and we can move on with our lives

Reply:
Done